xform.py

```diff
@@ -0,0 +1,61 @@
+import csv
```

Comment:
How does this file fit in? Seems a bit weird to have it floating around in the root of the repo.